html5/shared/BroadcastChannel.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @fileOverview
+ * The polyfill of BroadcastChannel API.
+ * This api can be used to achieve inter-instance communications.
+ *
+ * https://html.spec.whatwg.org/multipage/comms.html#broadcasting-to-other-browsing-contexts
+ */
+
+const channels = {}
```

Comment:
Well I think it may lead to memory leaks because we have no chance to clear registered `channels` if the developers do not close its `BroadcastChannel` instance.

It's better to have a check when `destroyInstance` called in js runtime or frameworks.